src/EnergyPlus/SetPointManager.cc

```diff
@@ -3703,6 +3787,253 @@ void GetSetPointManagerInputData(EnergyPlusData &state, bool &ErrorsFound)
+                                                                 NumNums,
+                                                                 IOStat,
+                                                                 lNumericFieldBlanks,
+                                                                 lAlphaFieldBlanks,
+                                                                 cAlphaFieldNames,
+                                                                 cNumericFieldNames);
+        UtilityRoutines::IsNameEmpty(state, cAlphaArgs(1), cCurrentModuleObject, ErrorsFound);
+
+        auto &setpointManager = state.dataSetPointManager->SystemNodeResetSetPtMgr(SetPtMgrNum);
+
+        setpointManager.Name = cAlphaArgs(1);
+        setpointManager.ctrlVarType = cAlphaArgs(2);
+
+        int typeNum = getEnumerationValue(SetPointManager::controlTypeNameUC, cAlphaArgs(2));
+        setpointManager.CtrlTypeMode = static_cast<SetPointManager::CtrlVarType>(typeNum);
+        if (setpointManager.CtrlTypeMode == SetPointManager::CtrlVarType::Invalid) {
+            // should not come here if idd type choice and key list is working
+            ShowSevereError(state, format("{}: {}=\"{}\", invalid field.", RoutineName, cCurrentModuleObject, cAlphaArgs(1)));
+            ShowContinueError(state, "..invalid " + cAlphaFieldNames(2) + "=\"" + cAlphaArgs(2) + "\".");
+            ShowContinueError(state, "..Valid values are \"Temperature\", \"MaximumTemperature\", or \"MinimumTemperature\".");
+            ErrorsFound = true;
+        }
+
+        setpointManager.SpAtLowRef = rNumericArgs(1);
+        setpointManager.SpAtHighRef = rNumericArgs(2);
+        setpointManager.LowRef = rNumericArgs(3);
+        setpointManager.HighRef = rNumericArgs(4);
+
+        setpointManager.RefNodeNum = GetOnlySingleNode(state,
+                                                       cAlphaArgs(3),
+                                                       ErrorsFound,
+                                                       DataLoopNode::ConnectionObjectType::SetpointManagerSystemNodeResetTemperature,
+                                                       cAlphaArgs(1),
+                                                       DataLoopNode::NodeFluidType::Blank,
+                                                       DataLoopNode::ConnectionType::Sensor,
+                                                       NodeInputManager::CompFluidStream::Primary,
+                                                       ObjectIsNotParent);
+
+        setpointManager.CtrlNodeListName = cAlphaArgs(4);
+        if (setpointManager.HighRef < setpointManager.LowRef) {
+            ShowWarningError(state, format("{}: {}=\"{}\", invalid field.", RoutineName, cCurrentModuleObject, cAlphaArgs(1)));
+            ShowContinueError(state,
+                              format("...{}=[{:.1R}] is less than {}=[{:.1R}].",
+                                     cNumericFieldNames(3),
```

Comment:
Shifted the field names in the error message. And noticed a typo in one of the field names. Fixes that in the idd.